src/software/ai/hl/stp/play/shoot_or_pass/shoot_or_pass_play_fsm_test.cpp

```diff
@@ -22,3 +22,103 @@ TEST(ShootOrPassPlayFSMTest, test_transitions)
 
     EXPECT_TRUE(fsm.is(boost::sml::state<ShootOrPassPlayFSM::AttemptShotState>));
 }
+
+TEST(ShootOrPassPlayFSMTest, DISABLED_test_should_abort_pass_guard)
```

Comment:
If this test fails and it will not be enabled as a part of this PR, you should tag the ticket with a TODO comment to it